### PR TITLE
updated to add the new 3.0 scope hoisting flag

### PR DESCRIPTION
**Pull request by maraisr**

### CODE REVIEW

**Comment by simon04** [comment]
For the reference, this documents https://github.com/webpack/webpack/pull/4951.

File: content/api/cli.md [hunk 1]
```diff
@@ -237,24 +237,25 @@ These allow you to configure the webpack [resolver](/configuration/resolve/) wit
...
+| --display-modules              | Display even excluded modules in the output                        | boolean |
+| --display-origins              | Display origins of chunks in the output                            | boolean |
+| --display-reasons              | Display reasons about module inclusion in the output               | boolean |
+| --display-used-exports         | Display information about used exports in modules (Tree Shaking)   | boolean |
+| --hide-modules                 | Hides info about modules                                           | boolean |
+| --sort-assets-by               | Sorts the assets list by property in asset                         | string  |
+| --sort-chunks-by               | Sorts the chunks list by property in chunk                         | string  |
+| --sort-modules-by              | Sorts the modules list by property in module                       | string  |
+| --verbose                      | Show more details                                                  | boolean |
+| --display-optimization-bailout | Scope hoisting fallback trigger                                    | boolean |
```
Comment (simon04) [inline content/api/cli.md]: Please add "(since webpack 3.0.0)". Good otherwise 👍.

Comment (simon04) [inline content/api/cli.md]: Oh yes, and please restore the alphabetical ordering.

Comment (maraisr) [inline content/api/cli.md]: Cool mate - would love too.

**Pull request merged**